openstack/fedora-35-x86_64/main.tf

```diff
@@ -0,0 +1,11 @@
+module "openstack" {
+  source = "../_base"
+
+  name      = "fedora-35"
+  image_id  = "d62a2182-947b-421e-b3db-9b55b4dda522"
+  flavor_id = "893c20cf-d5ea-4c7d-9eee-2bc4b3e5723e"
```

Comment:
I think this is the wrong `flavor_id` for RHOS-01.  Should be `f2c4469b-f516-46d1-8b87-1dcca68fb3d9`